### PR TITLE
Update Muon user documentation

### LINKED ISSUE #27839: Update user docs for muon
We need to update the user docs to relate the recent changes to plotting and fitting in the muon and FDA GUIs.

### PR DESCRIPTION
**Pull request by StephenSmith25**
**Description of work.**
This PR updates the user documentation for the Muon Analysis 2 and Frequency Domain Analysis GUIs.

**To test:**
Build docs-html and check the documentation pages Muon Analysis 2 and Frequency Domain Analysis.

Fixes #27839. 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: docs/source/interfaces/muon_fitting_tab.rst [hunk 1]
```diff
@@ -5,61 +5,32 @@ Fitting Tab
 
 .. image::  ../images/muon_interface_tab_fitting.png
    :align: right
-   :height: 500px
+   :height: 400px
 
-This tab is for performing fits on the data.
+This tab is used to evaluate a single fit object, which corresponds to a list of input workspaces and a fit function.
```
Comment (Mantid-Matthew) [inline docs/source/interfaces/muon_fitting_tab.rst]: Coudl be make clear this is a list of one or more workspaces here.

File: docs/source/interfaces/muon_sequential_fitting_tab.rst [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+**Fit quality** The quality of the fit returned from the fitting algorithm.
+
+**List of fit parameters** The columns that proceed this correspond to the values of the fit parameters. The value of these
+parameters can be edited. 
+
+
+Used By
+^^^^^^^
+
+:ref:`Muon Analysis 2 <MuonAnalysis_2-ref>`
```
Comment (Mantid-Matthew) [inline docs/source/interfaces/muon_sequential_fitting_tab.rst]: Again could we call it Muon Analysis

**Review by Matthew-Andrew — COMMENTED** [review]
Just a few minor comments

**Review by Matthew-Andrew — APPROVED** [review]
Looks good to be, good Job 👍

**Pull request merged**